### ISSUE TITLE
Setup zuul

### ISSUE THREAD

**Issue by lachmanfrantisek** [packit-service/dashboard#13]
- [ ] get inspired by the other projects in the packit-service organisation
- [ ] look at the docs (https://github.com/packit-service/research/tree/master/zuul)
- [ ] create necessary files (like we have in other repos)

What will be run in the pipelines:
  - have pre-commit as a first job (#12 ) (`pre-commit run --all`)
  - have some tests as a second job (https://github.com/packit-service/dashboard/issues/21)

**Comment by lachmanfrantisek** [comment]
I've added the missing link. (We can start working on it.)

**Comment by shreyaspapi** [comment]
I can work on this issue.

**Comment by lachmanfrantisek** [comment]
@shreyaspapi Nice!

**Comment by IceWreck** [comment]
@lachmanfrantisek 
Since he is working on the zull part, can I work on the tests part?

**Comment by lachmanfrantisek** [comment]
> @lachmanfrantisek
> Since he is working on the zull part, can I work on the tests part?

I'll remove the testing part from this issue and create a new issue for tests so we can track that -- @shreyaspapi, is that ok for you?

**Comment by shreyaspapi** [comment]
@lachmanfrantisek yeah, that would be fine.

**Comment by lachmanfrantisek** [comment]
I've split the task and updated the docs.

**Comment by shreyaspapi** [comment]
Please don't merge until I do the necessary changes to https://softwarefactory-project.io/r/config/

**Comment by shreyaspapi** [comment]
@lachmanfrantisek Anything else needed here?

**Comment by lachmanfrantisek** [comment]
@shreyaspapi probably nothing, thanks!
